### ISSUE TITLE
New rake cannot be installed with expected ruby version

### ISSUE THREAD

**Issue by viraptor** [envato/ami-spec#33]
Running `bundle install` in the fresh checkout results in:

```
Installing rake 12.3.1
Gem::InstallError: rake requires Ruby version >= 2.0.0.
...
An error occurred while installing rake (12.3.1), and Bundler cannot continue.
Make sure that `gem install rake -v '12.3.1'` succeeds before bundling.
```
`.ruby-version` requires `1.9.3p551`, which is quite dated now. There's a commit about the downgrade from 2.2 (https://github.com/envato/ami-spec/commit/a3180c0fd0b416fe20faae1549afdc53ef039db7) but without an explanation about why it's done.

Is there something preventing a bump to the latest ruby 2.5? After manual upgrade, tests pass.

**Comment by patrobinson** [comment]
We did it for a reason, what that reason was I cannot recall. What version of ruby does the agent running these use?

**Comment by viraptor** [comment]
Not sure - it doesn't publish it as metadata. It's running in `queue=packer-build`.

**Comment by viraptor** [comment]
Actually there's a warning with the ruby path included. It's using `2.3.x`

**Comment by patrobinson** [comment]
Let's move to 2.3 or higher then

**Comment by patrobinson** [comment]
Fixed in version 1.2.0
